samples/hybrid-app-sample/main.gd

```diff
@@ -59,9 +59,10 @@ func _process(_delta: float) -> void:
 		panel_switcher_layer.update_pointer(t.origin, -t.basis.z, pointer_pressed)
 
 func _on_right_controller_pointer_button_pressed(p_name: String) -> void:
-	if p_name == 'trigger_click':
+	print("button: ", p_name)
+	if p_name == 'trigger_click' or p_name == 'select_button':
```

Comment:
I'm not sure what's going on with this. I took a quick peek at the action map, and it seemed like I should be getting "trigger_click" for Pico 4 controllers, but instead I'm getting "select_button". Needs to be investigated more.

Reply:
Could this be related to the issue mentioned in RC, where Pico defaults to a compat mode?

Reply:
Yep, I think it's the same thing, but I haven't had a chance to investigate further yet.

But we'll definitely want to have the action map working correctly before taking this out of draft.

Reply:
It looks like there is a new controller interaction profile for Pico 4 Ultra (`/interaction_profiles/bytedance/pico4s_controller` - note the `s` in `pico4s`) and that is somehow related to it falling back on "Simple controller", because if I patch that into Godot, I'm able to get controller input normally. I'll make a Godot PR later, but I haven't yet managed to track down documentation on that interaction profile, so it'll have to be a draft until we can confirm all the paths

Reply:
See https://github.com/godotengine/godot/pull/103234